Update Readme and make maxperf default for reth build (#38)
LiskHQ/lisk-node: README.md

## Usage

> **Note**:

> > - It is currently not possible to run the nodes with the `--op-network` flag until the open PRs for adding configs for Lisk [Sepolia](https://github.com/ethereum-optimism/superchain-registry/pull/506) and Lisk [Mainnet](https://github.com/ethereum-optimism/superchain-registry/pull/502) are merged into the [superchain-registry](https://github.com/ethereum-optimism/superchain-registry)

Reviewer: ```suggestion
> - It is currently not possible to run the nodes with the `--op-network` flag until the open PRs for adding configs for [Lisk Sepolia](https://github.com/ethereum-optimism/superchain-registry/pull/506) and [Lisk Mainnet](https://github.com/ethereum-optimism/superchain-registry/pull/502) are merged into the [superchain-registry](https://github.com/ethereum-optimism/superchain-registry).
```

> > - It is currently not possible to run the nodes with the `--op-network` flag until the open PR's to add configs for Lisk [Sepolia](https://github.com/ethereum-optimism/superchain-registry/pull/506) and Lisk [Mainnet](https://github.com/ethereum-optimism/superchain-registry/pull/502) have been merged to the [superchain-registry](https://github.com/ethereum-optimism/superchain-registry)

Reviewer 2: ```suggestion
> - It is currently not possible to run the nodes with the `--op-network` flag until the open PRs for adding configs for Lisk [Sepolia](https://github.com/ethereum-optimism/superchain-registry/pull/506) and Lisk [Mainnet](https://github.com/ethereum-optimism/superchain-registry/pull/502) are merged into the [superchain-registry](https://github.com/ethereum-optimism/superchain-registry)
```
